index.html

```diff
@@ -5,27 +5,41 @@
     <title>IMDB Top 250</title>
     <!--link rel="stylesheet" href="/node_modules/bootstrap/dist/css/bootstrap.css"-->
     <link rel="stylesheet" type="text/css" href="/node_modules/angular-material/angular-material.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,400italic">
     <link rel="stylesheet" href="css/styles.css" media="screen" title="no title" charset="utf-8">
   </head>
   <!-- enter app name in ng-app -->
   <body ng-app="imdbApp">
-
+    <!--nav bar-->
+    <div ng-controller="navbarController">
+      <md-content class="md-padding">
+        <md-nav-bar md-selected-nav-item="currentNavItem" nav-bar-aria-label="navigation links">
```

Comment:
Excellent use of the navbar to show across all pages! Another way to abstract out this logic is to put it inside its own template. You can then reference the template using "ng-include"